docs/background/basis/README.md

```diff
@@ -64,6 +64,21 @@
      - :class:`~nemos.basis.OrthExponentialEval`  :raw-html:`<br />`
        :class:`~nemos.basis.OrthExponentialConv`
      - 🟢 Eval
+   * - **Identity Function**
```

Comment:
Feels odd to have these two be separate, as they're then the only ones that don't have both modes. Why not include them in the same row?

Reply:
My only point here is that history is like a lagged version of the identity, so it is not exactly the "conv" version of identity. 

If it was that, i would expect that the kernel  for convolution would be just a single `np.linsapce(0,1,n_samples)` (which is f(x) = x when x is the linspace).

Instead the history basis is a bunch of delta function that peaks at different lags. 

Also the image in the table of identity doesn't apply to the history so we would not be able to have an image that makes sense for both.

What do you think?

Reply:
I think that makes sense.